### ISSUE TITLE
Bug: Incorrect pagination dot count

### ISSUE THREAD

**Issue by ralphchristianeclipse** [SSENSE/vue-carousel#24]
### Reproduction Code

```html
        <carousel :perPage="1" autoplay scrollPerPage easing="ease-in-out">
            <slide>a</slide>
         
            <slide>b</slide>

        </carousel>
```

### What is expected?

Must show 2 pagination dots and will not slide on a blankpage

### What happened?
![image](https://cloud.githubusercontent.com/assets/25447231/23336859/6aa4e9b0-fc17-11e6-8548-dcab9e967dd5.png)
![image](https://cloud.githubusercontent.com/assets/25447231/23336872/be23eab4-fc17-11e6-8c2d-e8951b24d88a.png)

**Comment by toddlawton** [comment]
Hi @ralphchristianeclipse, could you please confirm that you aren't configuring your templates the same way that #20 is?

**Comment by ralphchristianeclipse** [comment]
@toddlawton 

im using vue-loader with webpack environment

so i'm using templates for that

**Comment by toddlawton** [comment]
@ralphchristianeclipse I can't seem to reproduce that in the vue-play dev environment.
Would it be possible to send a codepen or jsfiddle?

**Comment by abishekrsrikaanth** [comment]
@toddlawton You should be able to reproduce with http://codepen.io/abishekrsrikaanth/pen/dvMayg. I am having the same issue as well

**Comment by toddlawton** [comment]
@ralphchristianeclipse I found a solution and released it in [0.6.2](https://github.com/SSENSE/vue-carousel/releases/tag/0.6.2). Cheers!
